src/models/operation.ts

```diff
@@ -15,13 +15,17 @@ class Operation {
     address: string;
     amount: number;
 
-    // self sent (sent to non-change address belonging to same xpub)
+    // self sent (sent to same address)
     self: boolean; 
 
+    // sent to non-change address belonging to same xpub
+    sentToSibling: boolean;
+
```

Comment:
you could create an enum describing the the operation type

Reply:
Done.